php/router.php

```diff
@@ -0,0 +1,105 @@
+<?php
+// Used by `wp server` to route requests.
```

Comment:
It would be great to have proper PHPdoc for everything in this file, given we're exposing them as top-level functions and not private methods on the `Server_Command` class.